extensions/amp-video-iframe/1.0/storybook/Basic.amp.js

```diff
@@ -1,44 +1,61 @@
 import {withAmp} from '@ampproject/storybook-addon';
-import {boolean, number, text, withKnobs} from '@storybook/addon-knobs';
 
 import * as Preact from '#preact';
 
 export default {
   title: 'amp-video-iframe-1_0',
-  decorators: [withKnobs, withAmp],
+  decorators: [withAmp],
   parameters: {
     extensions: [{name: 'amp-video-iframe', version: '1.0'}],
     experiments: ['bento'],
   },
+  args: {
+    width: '640px',
+    height: '360px',
+    ariaLabel: 'Video Player',
```

Comment:
nit: use `aria-label` so you don't have to extract it and rename it in the fixtures

Reply:
using a hyphen in an arg name gives me a linting error, i dont think i can get around extracting and renaming in this case.

Reply:
you can turn off the rule with an eslint-disable-next-line comment otherwise feel free to merge